testsuite/tests/ui/conftest.py

```diff
@@ -387,8 +388,10 @@ def set_callback_urls(auth0_client):
     Set callback urls for Auth0 application
     """
 
+    @backoff.on_predicate(backoff.constant, lambda x: x['callbacks'] == [], 10)
```

Comment:
sleep(3) was replaced by constant backoff of count 10? doesn't look equal (plus missing jitter=None ...)